ZupIT/ritchie-formulas
# Machine learning template formula

**- What I did**
Added formula to create a jupyter notebook template for diverse machine learning scenarios

**- How I did it**
Created a python formula and tried to mimic [this cheat cheet decision tree](https://scikit-learn.org/stable/tutorial/machine_learning_map/index.html). Based on the user response, a `.ipynb` is dynamically generated and written to the `pwd`

**- How to verify it**
Load the formula and run `jupyter machine_learning template`, after a few question about the nature of your data you should see a notebook created on the same directory you are currently on

**- Description for the changelog**
Added a new formula to create machine learning based jupyter notebooks

![Screen Recording 2020-07-09 at 00 06 58](https://user-images.githubusercontent.com/3487122/86992962-1ffb7c00-c179-11ea-87b7-42730bf074cd.gif)
